blackbox/docs/configuration/cluster.txt

```diff
@@ -587,6 +587,23 @@ Routing Allocation
    recover their unassigned local primary shards immediatelly after restart, in
    case the ``recovery.initial_shards`` setting is satisfied.
 
+.. _cluster.routing.rebalance.enable:
+
+**cluster.routing.rebalance.enable**
```

Comment:
maybe skip the `enable` suffix? There is no on off. Or rename enable to `scope`? Just a suggestion.

Reply:
@mxm IMHO since this is an ES setting I would stay with the same name here to be also consistent with the other ES settings that we expose.